postgres-appliance/Dockerfile

```diff
@@ -56,6 +73,10 @@ RUN export DEBIAN_FRONTEND=noninteractive \
                 pgxn install $extension; \
             done \
 
+            && if [ "$version" != "9.3" ]; then \
+                make -C postgres-decoderbufs-$DECODEBUFS_VER clean install; \
```

Comment:
Yeah, the code was written by someone who probably thought C++ is just C without classes. Most of those are stupid declaration not preceding the code warnings, the pointer init in set_datum_value is harmless, as it is never used before being initialized again, and the is_rel_non_selective is only used in this context:
``
bool is_rel_non_selective;
 const char *selectiveInfo = is_rel_non_selective ? "non selective" : "selective";
``
and then initialized properly, so it's not a problem apart from the wrong debug message.

On the other hand, this indicates the code is rough (as it doesn't have any issues on github, so probably no-one used it yet), so we should tread lightly.